resources/manifest_template.json5

```diff
@@ -0,0 +1,65 @@
+{
+  /* the title which lists the project
+   * required */
+  "title": "test_project0",
+  /* put an url here or multiple that are representative for the project
+   * required*/
+  "url": [
+    "www.github.com/knex/"
+  ],
+  /* put multiple anlalysis links here */
+  "analysis": [
+    {
+      "url": "here-you-find-the-analysis.com",
+      "description": "thats what this analysis is about"
+    }
+  ],
+  /* a list of authors and their email addresses
+   * at least one */
+  "authors": [
+    {
+      "name": "test_owner0",
+      "email": "email@address.com"
+    }
+  ],
+  /* the team name  */
+  "team": {
+    "team_name": "teamname"
+  },
+  "status": "DONE",
+  /* a short description*/
+  "description": "can this be a multiline description
```

Comment:
i'm kind of confused here because it seams to work without them at least wenn using advanced REST client but maybe it is inserting them by default